Sources/Target/Darwin/Thread.cpp

```diff
@@ -33,7 +33,88 @@ namespace Darwin {
 using Host::Darwin::LibProc;
 
 Thread::Thread(ds2::Target::Process *process, ThreadId tid)
-    : super(process, tid), _lastSyscallNumber(-1) {}
+    : super(process, tid), _lastSyscallNumber(-1) {
+  //
+  // Initially the thread is stopped.
+  //
+  _resumeWithPtrace = true;
```

Comment:
Use initializer list for this and add a comment that explains why the first resume should be done with ptrace.